### PR TITLE
Feature/sphinx cicd

### PR DESCRIPTION
**Pull request by vpopil**
Sphinx updates. README documentation primarily moved to Sphinx.

### CODE REVIEW

File: .github/workflows/deploy_sphinx_docs.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Sphinx Docs Deploy
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
```
Comment (djsauble) [inline .github/workflows/deploy_sphinx_docs.yml:6]:
```suggestion
# events but only for the dev branch
```

File: .github/workflows/deploy_sphinx_docs.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        #pre-build-command: # optional
+
+    - name: Commit documentation changes
+      run: |
+        git clone https://github.com/pkgpkr/Package-Picker.git --branch gh-pages --single-branch gh-pages
+        cd gh-pages
+        rm -r *
+        cp -r ../docs/build/html/* .
+        touch .nojekyll
+        git config --local user.email "action@github.com"
```
Comment (djsauble) [inline .github/workflows/deploy_sphinx_docs.yml:49]: Does it make sense to use our shared `pkgpkr1` account for this?

File: docs/source/setup.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+```
+AWS_ACCESS_KEY_ID     # Your AWS account's access key
+AWS_SECRET_ACCESS_KEY # Your AWS account's secret access key
+CLIENT_ID             # ID of the GitHub app used by the web server
+CLIENT_SECRET         # Secret for the GitHub app
+DB_HOST               # Database URL
+DB_PASSWORD           # Database password
+DB_USER               # Database user
+DOMAIN_NAME           # The domain name where the site is hosted (e.g. http://pkgpkr.com)
```
Comment (djsauble) [inline docs/source/setup.md:14]:
```suggestion
DOMAIN_NAME           # The domain name where the site is hosted (e.g. https://pkgpkr.com)
```

File: docs/source/setup.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+MONTH                 # How many months of data to scrape
+SELENIUM_TEST         # Set if running Selenium tests
```
Comment (djsauble) [inline docs/source/setup.md:16]:
```suggestion
SELENIUM_TEST         # Set to 1 (or any value) if running Selenium tests
```

File: .github/workflows/deploy_sphinx_docs.yml (hunk 2 above)
Comment (vpopil) [reply .github/workflows/deploy_sphinx_docs.yml:49]: @djsauble it is using that account's token to do that commit, but this was a clean way to specify that it's committed by GithubAction..  idk, I kind of like as is, but we could plug in another name and email if you'd like, I'm cool with either. but technically it's just metadata on the commit

Comment (djsauble) [reply .github/workflows/deploy_sphinx_docs.yml:49]: Naw, this is fine. I've seen some companies have a "bot" user for this purpose, maybe we spin up a `bot@pkgpkr.com` email eventually.

Comment (vpopil) [reply .github/workflows/deploy_sphinx_docs.yml:49]: Oh i see, sure, why not, maybe between 5/6 and 5/14 : )

**Pull request merged**